src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10994,6 +11176,12 @@ private bool ShouldParseLambdaParameterType(bool hasModifier)
                 return true;
             }
 
+            // If this is a function pointer type
```

Comment:
comment not useful :)

Reply:
I'm just following the rest of the style of the function.

---
In reply to: [348022338](https://github.com/dotnet/roslyn/pull/39879#discussion_r348022338) [](ancestors = 348022338)

Reply:
If I need to push another revision, I'll remove this PR. Otherwise, I'll followup later

---
In reply to: [348022338](https://github.com/dotnet/roslyn/pull/39879#discussion_r348022338) [](ancestors = 348022338)